homeworks/becseipeter/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,79 @@
+class Car:
+    
+    def __init__(self, brand: str, model: str, year: int):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = 0                    
+        self.fuel_level = 100               
+
+    def drive(self, kilometers):
+        fuel_needed = kilometers * .1
+        if self.fuel_level > fuel_needed:
+            self.fuel_level -= fuel_needed
+            self.mileage += kilometers
+        else:
+            print(f'Find a gas station.')
+
+    def refuel(self, amount):
+        if self.fuel_level + amount > 100:
+            self.fuel_level = 100
+        else:
+            self.fuel_level += amount
+            print(f"{self.brand} {self.model} refueled. Fuel level is now {self.fuel_level}%.")
+
+
+class Fleet:
+
+    def __init__(self, name):
+        self.name = name
+        self.cars = []  
+
+    def add_cars(self, car):
+        self.cars.append(car)
+
+    def remove_car(self, car: Car):
+        for car in self.cars:
```

Comment:
ez így nem azt csinálja amit te szeretnél, a car így a for loop-ban létrehozott car lesz, nem a paraméter amit átadtál.

így mehetsz tutira hogy azt törlöd amit kell:

```
if car in self.cars:
    self.cars.remove(car)
```